tgui/packages/tgui/interfaces/common/AtmosControls.tsx

```diff
@@ -112,7 +142,7 @@ export const Vent = (props, context) => {
 };
 
 
-export const Scrubber = (props, context) => {
+export const Scrubber = (props:{scrubber:ScrubberData}, context) => {
```

Comment:
Refactor that type into `type ScrubberProps`, this is how many ts devs do it, inlining is usually a horrible idea.

Reply:
Will keep that in mind. Let me know if b1a7442d5ea1e0cd73274e67069b217cdee8ca3f is bad.